kasa/iot/iotdevice.py

```diff
@@ -720,6 +730,10 @@ def _get_device_type_from_sys_info(info: dict[str, Any]) -> DeviceType:
                 return DeviceType.LightStrip
 
             return DeviceType.Bulb
+
+        if "camera" in type_.lower():
+            return DeviceType.Camera
```

Comment:
```suggestion
        if "system" in sys_info:
            return DeviceType.Camera
```
And move to the top of the method.

Reply:
Yes I think so